app/google-map/google-map.service.ts

```diff
@@ -21,11 +23,11 @@ export class GoogleMapService {
 
         options = Object.assign(this.options, options || {});
 
-        this.gMap = new google.maps.Map(document.getElementById("google-map"), options);
+        this.gMap = new google.maps.Map(document.getElementById(googleMapID), options);
```

Comment:
how about passing googleMap element as a variable? that will decouple service from component

by the way it's not a common practice to use ```document``` to query elements - you can use ```@ViewChild``` in your component for it, as a result you will get rid of a ```googleMapID``` constant